Makefile

```diff
@@ -2,7 +2,7 @@
 EPOCH_TEST_COMMIT	:= 78e6667ae2d67aad100b28ee9580b41b7a24e667
 OUTPUT_DIRNAME		?= output/
 DOC_FILENAME		?= oci-runtime-spec
-SHELL			?= $(shell command -v bash 2>/dev/null)
+SHELL			?= $(shell command -v sh 2>/dev/null)
```

Comment:
I don't think we need this.  From the [GNU Make docs][1]:

> The program used as the shell is taken from the variable `SHELL`. If this variable is not set in your makefile, the program `/bin/sh` is used as the shell.

Although GNU Make does some dancing for DOS/Windows.  Are you including this line to skip that dance?  If so, why?

[1]: https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html